cli/napi/threadsafe_functions.rs

```diff
@@ -9,6 +9,12 @@ use std::ptr::NonNull;
 use std::sync::atomic::AtomicUsize;
 use std::sync::Arc;
 
+#[repr(transparent)]
+pub struct SendPtr<T>(pub *const T);
```

Comment:
*"question:** Should this be crate scoped?

Reply:
Probably yeah. CLI is a bin crate so doesn't really matter?